### DISCUSSION TITLE
How to interpret the number of lines in "*.shuffled.00*.out" file

### DISCUSSION THREAD

**Discussion by LeiLiu8102** [deepmodeling/dpgen#410] [Q&A]
Hi, thank you for developing this package, which is very useful.

In the `run` step of dpgen, I found that it generates three files (in `02.fp`) named `res_accurate.shuffled.000.out`, `res_failed.shuffled.000.out` and `candidate.shuffled.000.out`, each containing some configurations. Does it mean that the configurations (or structures) in `res_accurate.*` belong to the neighborhood of the training data, and the ones in `res_failed.shuffled.*.out` are far from the training data? Then what about the configurations in `candidate.shuffled.*.out`? How could I know how many of the generated configurations in DPMD simulations are "new" configurations in '01.model_devi' step?

The other question is about the exploration strategy. In the iterations reported in your paper (as shown in the attached figure), you didn't increase the temperatures of DPMD simulations until the labeling percentage approaches 0. Is there any reason behind this strategy? Is this better than exploring temperatures randomly?

Appreciate it again.

**Comment by LeiLiu8102** [comment]
It seems that the figure is not attached. You can see the Table S1 in the supplementary material of this article: https://journals.aps.org/prmaterials/abstract/10.1103/PhysRevMaterials.3.023804

**Comment by njzjz** [comment]
> Does it mean that the configurations (or structures) in res_accurate.* belong to the neighborhood of the training data, and the ones in res_failed.shuffled.*.out are far from the training data? Then what about the configurations in candidate.shuffled.*.out? How could I know how many of the generated configurations in DPMD simulations are "new" configurations in '01.model_devi' step?

It has been explained in Section 2 of https://doi.org/10.1016/j.cpc.2020.107206.
